site2/docs/client-libraries-python.md

```diff
@@ -62,7 +62,7 @@ The complete Python API reference is available at [api/python](/api/python).
 
 ## Examples
 
-You can find a variety of Python code examples for the `pulsar-client` library.
+You can find a variety of Python code examples for the [pulsar-client](/pulsar-client-cpp/python) library.
```

Comment:
```suggestion
You can find a variety of Python code examples for the [pulsar-client](/pulsar-client-cpp/python/examples) library.
```

Reply:
In the following examples are /pulsar-client-cpp/python/test_consumer.py and /pulsar-client-cpp/python/test_test_producer.py, soI don't think it's necessary to change it here。Or we should move these two files to /examples